app/controllers/organizations/tasks_controller.rb

```diff
@@ -40,10 +58,10 @@ def update
           Organizations::TaskService.new(@task).create_next
         end
 
-        format.turbo_stream { render turbo_stream: turbo_stream.replace("tasks_list", partial: "organizations/pets/tasks/tasks", locals: {task: @task}) }
+        format.turbo_stream { render turbo_stream: turbo_stream.replace("tasks_pet_#{@pet.id}", partial: "organizations/pets/tabs/tasks", locals: {task: @task}) }
```

Comment:
We want to broadcast to the specific task, not the whole tab. Instead of this, I think you could remove all that and have this:
```ruby
# app/views/organizations/tasks/update.turbo_stream.erb
<%= turbo_stream.replace @task do %>
  <%= render @task %>
<% end %>
```
Now, you should be able to open multiple edits on the index, and click "Update Task" on one, and the rest will stay open.

Reply:
I was going off the prior implementation and only updated the target to use the new dom id. Happy to change this, the only consideration is that the task list will no longer update to the correct order until a refresh.

For example, the task list order is overdue, upcoming then completed. If you change the first overdue task to a future date it will still be the first on the list until a refresh. I can see pro's and cons to both ways.

Reply:
Thanks, @jmilljr24 ! I did not consider that ramification. 

You are right, as I am getting ambitious past the original intent of the PR... I am a new collaborator and will try to work on focusing on scope on my future reviews. Sorry!

I think we can leave this as is then based on your points, unless you have already made my suggested changes. If we want to clean this up more, we can do so in a future PR.

Reply:
I haven't touched anything yet. Love your insight thought and happy to do whatever is decided on!! If you don't mind, could you please indicate on the other revisions requested so there is no ambiguity on what you still want changed since the conversations are starting to overlap.